odl/solvers/functional/functional.py

```diff
@@ -808,9 +809,9 @@ def __init__(self, func, linear_term):
         Parameters
```

Comment:
Further up: single ticks for `Functional`, and the word "representing" can be removed.